.github/actions/setup-dependencies/action.yml

```diff
@@ -13,35 +13,13 @@ runs:
       run: sudo unblob/install-deps.sh
       shell: bash
 
-    - name: Setup Python
-      uses: actions/setup-python@v5
+    - name: Install uv & Python
```

Comment:
Did a quick search if any `poetry` stuff remained, the things I found:
* `flake.lock`: `poetry2nix`  - don't we need to sync the lock file?
* `bug-report-🐞.md` 
* `build-publish-image.yml` - actually does some `poetry build` stuff

Reply:
We don't use poetry2nix, so it is some garbage.
Somewhere I lost the other changes, hmm.

Reply:
poetry2nix is a transitive dependency of devenv